vault/resource_transit_secret_backend_key.go

```diff
@@ -156,6 +156,22 @@ func transitSecretBackendKeyResource() *schema.Resource {
 				Computed:    true,
 				Description: "Whether or not the key supports signing, based on key type.",
 			},
+			// import only options
+			"ciphertext": {
```

Comment:
I wonder if it would make sense to also add a `bool` field that denotes that this `key` is an import/foreign? Could be overkill,  but it would be more explicit.